### PR TITLE
I added dropcaps to AMP and you'll never guess what happened next...*

### PR DESCRIPTION
**Pull request by zeftilldeath**
screen shot 2015-10-02 at 17 12 18

*Buzzfeed style heading for added intrigue.

### CODE REVIEW

**Comment by johnduffell** [comment]
:+1: great! :speedboat:

File: common/app/views/fragments/amp/stylesheets/main.scala.html [hunk 1]
```diff
@@ -193,6 +190,27 @@
...
+        line-height: 2.5rem;
+    }
+
+    .drop-cap {
+        float: left;
+        display: inline-block;
+        text-transform: uppercase;
+        -webkit-box-sizing: border-box;
+        -moz-box-sizing: border-box;
+        box-sizing: border-box;
```
Comment (sndrs) [inline common/app/views/fragments/amp/stylesheets/main.scala.html]: we auto-prefix css on the main site, we need to look at how CSS is written for AMP, we don't really want to two systems for two different sites... cc @johnduffell @NataliaLKB @stephanfowler

Comment (stephanfowler) [inline common/app/views/fragments/amp/stylesheets/main.scala.html]: @sndrs agreed. We're just rushing something out of the door for this week's preview. We'll need to revisit it all, afterwards.

**Comment by johnduffell** [comment]
@sndrs I'd love to involve you in setting something sustainable for building the CSS, what we have now mostly happened by accident and lack of experience.  After preview if you have time we'd love to have some time with you to get something going?

**Comment by sndrs** [comment]
certainly

On Mon, 5 Oct 2015 at 14:20 John Duffell <notifications@github.com> wrote:

> @sndrs <https://github.com/sndrs> I'd love to involve you in setting
> something sustainable for building the CSS, what we have now mostly
> happened by accident and lack of experience. After preview if you have time
> we'd love to have some time with you to get something going?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/10758#issuecomment-145525509>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Pull request merged**